packages/pci-sdk-iframe/src/pages/app/components/Info/Info.tsx

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
+import { ITheme } from '../../../../types/IThemes';
 
 interface IInfoProps {
 	message: string;
+	theme: ITheme;
 }
 export default function Info(props: IInfoProps) {
+	const color = props.theme.container?.color;
```

Comment:
What happens when color is undefined? Is ignored by the style tag or is set to undefined? 🤔

Reply:
It's ignored.